app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -35,9 +35,8 @@
         {% include "toolkit/page-heading.html" %}
     {% endwith %}
     <p class="padding-bottom-small">All requirements are published on the Digital Marketplace where anyone can see them.</p>
-
-    <p><strong>Suppliers will be able to apply for:</strong></p>
-    <p class="padding-bottom-small">2 weeks ending at midnight.</p>
+    <p class="padding-bottom-small">All requirements are open for {{ dates.application_open_weeks }} weeks.</p>
+    <p class="padding-bottom-small">If you publish your requirements today ({{ dates.today }}), suppliers will be able to apply until {{ dates.closing_time }}, {{ dates.closing_date }}.</p>
```

Comment:
It might be nice to stick to our `datetimeformat` from utils rather than making an ad hoc display format.  I don't see why we'd want a different format here than other places where we show dates to users. (Maybe ask @ralph-hawkins if it really must be `{{ dates.closing_time }}, {{ dates.closing_date }}`?)